### ISSUE TITLE
Tasks using v1.3 fail when there are no linting errors

### ISSUE THREAD

**Issue by jessie-codes** [TrueBrain/actions-flake8#20]
I tested the same source code with `v1.2` and `v1.3`, and when using `v1.3`, my workflow fails even though there aren't any linting errors. I'm seeing output in the logs of `Flake8 found no problems`.

It appears that the issue is within the final lines of `entrypoint.sh` as you're using `$res` for the exit code, but that variable is no longer being set in the new version.

```bash
# If we are in warn-only mode, return always as if we pass
if [ -n "${INPUT_ONLY_WARN}" ]; then
    exit 0
else
    exit $res
fi
```

**Comment by TrueBrain** [comment]
Oops, I am so sorry ... I tested everything, but not an happy-flow .. I am going to add a test for that right now!

**Comment by TrueBrain** [comment]
Fixed in v1.4. Sorry again, and tnx for noticing so quickly :)

**Comment by jessie-codes** [comment]
No problem, thanks for the quick turn around on the fix! I just updated to `v1.4` and my workflows are back in business. :)
